recipes/m4rie/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "20150908" %}
+{% set name = "m4rie" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://bitbucket.org/malb/m4rie/downloads/{{ name }}-{{ version }}.tar.gz
+  sha256: a0c3f46c399ed2e4af5aaad9de9db0962d15d5ccb9bd0e313df6b2c16bf8e0b1
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libtool
+    - libpng >=1.6.28,<1.7
+    - zlib 1.2.*
+    - m4ri
```

Comment:
This will probably fail in the feedstock creation b/c of the race condition (`m4ri` will not be available yet). It is fine but you have to remember to restart the CIs once `m4ri` finishes building.